lib/prompts/downloadProjectPrompt.ts

```diff
@@ -1,26 +1,37 @@
-// @ts-nocheck
-const { promptUser } = require('./promptUtils');
-const { getAccountId } = require('@hubspot/local-dev-lib/config');
-const { fetchProjects } = require('@hubspot/local-dev-lib/api/projects');
-const { logError, ApiErrorContext } = require('../errorHandlers/index');
-const { EXIT_CODES } = require('../enums/exitCodes');
-const { i18n } = require('../lang');
+import { promptUser } from './promptUtils';
+import { getAccountId } from '@hubspot/local-dev-lib/config';
+import { fetchProjects } from '@hubspot/local-dev-lib/api/projects';
+import { logError, ApiErrorContext } from '../errorHandlers/index';
+import { EXIT_CODES } from '../enums/exitCodes';
+import { i18n } from '../lang';
+import { Project } from '@hubspot/local-dev-lib/types/Project';
 
 const i18nKey = 'lib.prompts.downloadProjectPrompt';
 
-const createProjectsList = async accountId => {
+type DownloadProjectPromptResponse = {
+  project: string;
+};
+
+async function createProjectsList(
+  accountId: number | null
+): Promise<Project[]> {
+  const account = accountId || undefined;
   try {
-    const { data: projects } = await fetchProjects(accountId);
+    const { data: projects } = await fetchProjects(accountId!);
```

Comment:
I don't think we should assert that `accountId` isn't null since we know it can be. It could make sense to make this function only accept a number, and throw an error in `downloadProjectPrompt` if accountId doesn't exist. That way you won't need both `accountId` and `account`, which is a little confusing.

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1288/commits/dbfad7214b180917cc226ffaee59f96b198bf5f3